### PR TITLE
update config

### PR DESCRIPTION
**Pull request by cydrain**
Signed-off-by: yudong.cai <yudong.cai@zilliz.com>

**What type of PR is this?**

api-change / bug / design / documentation / feature

**What this PR does / why we need it:**

**Which issue(s) this PR fixes:**

Fixes #

**Special notes for your reviewer:**

**Does this PR introduce a user-facing change?:**

**Additional documentation (e.g. design docs, usage docs, etc.):**

### CODE REVIEW

File: core/conf/demo/server_config.yaml [hunk 1]
```diff
@@ -46,9 +46,12 @@ server_config:
 #                      | loaded when Milvus server starts up.                       |            |                 |
 #                      | '*' means preload all existing tables.                     |            |                 |
 #----------------------+------------------------------------------------------------+------------+-----------------+
+# auto_flush_interval  | Interval of auto flush. Unit is millisecond.               | Integer    | 1000 (ms)       |
```
Comment (yamasite) [inline core/conf/demo/server_config.yaml]: The interval, in milliseconds, at which Milvus automatically flushes data to disk.

File: core/conf/demo/server_config.yaml [hunk 2]
```diff
@@ -165,3 +168,24 @@ gpu_resource_config:
 #----------------------+------------------------------------------------------------+------------+-----------------+
 tracing_config:
   json_config_path:
+
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# WAL Config           | Description                                                | Type       | Default         |
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# enable               | Switch of function WAL.                                    | Boolean    | false           |
```
Comment (yamasite) [inline core/conf/demo/server_config.yaml]: Whether to enable write-ahead logging (WAL) in Milvus. If WAL is enabled, Milvus writes all data changes to log files in advance before implementing data changes. WAL ensures the atomicity and durability for Milvus operations.

File: core/conf/demo/server_config.yaml [hunk 3]
```diff
@@ -165,3 +168,24 @@ gpu_resource_config:
...
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# recovery_error_ignore| Whether ignore the error which happens during WAL recovery | Boolean    | true            |
```
Comment (yamasite) [inline core/conf/demo/server_config.yaml]: Whether to ignore logs with errors that happens during WAL recovery. If true, when Milvus restarts for recovery and there are errors in WAL log files, log files with errors are ignored. If false, Milvus does not restart when there are errors in WAL log files.

File: core/conf/demo/server_config.yaml [hunk 4]
```diff
@@ -165,3 +168,24 @@ gpu_resource_config:
...
+#                      | stage.                                                     |            |                 |
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# buffer_size          | The size of the WAL buffer. Unit is MB.                    | Integer    | 256 (MB)        |
```
Comment (yamasite) [inline core/conf/demo/server_config.yaml]: Sum total of the read buffer and the write buffer in MBs. `buffer_size` must be in range [64, 4096] (MB) . If the value you specified is out of range, Milvus automatically uses the boundary value closest to the specified value. It is recommended you set `buffer_size` to a value greater than the inserted data size of a single insert operation for better performance.

File: core/conf/demo/server_config.yaml [hunk 5]
```diff
@@ -165,3 +168,24 @@ gpu_resource_config:
...
+#                      | It should be in range [64, 4096]. If the value set out of  |            |                 |
+#                      | the range, the system will use the boundary value.         |            |                 |
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# wal_path             | The root path of WAL relative files, include WAL meta      | String     | NULL            |
```
Comment (yamasite) [inline core/conf/demo/server_config.yaml]: Location of WAL log files.

File: core/conf/demo/server_config.yaml (hunk 1 above)
Comment (yamasite) [reply core/conf/demo/server_config.yaml]: Please apply this change to other files.

File: core/conf/demo/server_config.yaml (hunk 2 above)
Comment (yamasite) [reply core/conf/demo/server_config.yaml]: Please apply this change to other files.

File: core/conf/demo/server_config.yaml (hunk 3 above)
Comment (yamasite) [reply core/conf/demo/server_config.yaml]: Please apply this change to other files.

File: core/conf/demo/server_config.yaml (hunk 4 above)
Comment (yamasite) [reply core/conf/demo/server_config.yaml]: Please apply this change to other files.

File: core/conf/demo/server_config.yaml (hunk 5 above)
Comment (yamasite) [reply core/conf/demo/server_config.yaml]: Please apply this change to other files.

**Review by yamasite — CHANGES_REQUESTED** [review]

**Review by shengjun1985 — APPROVED** [review]

**Pull request closed without merging**